compiler-rt/lib/asan_abi/asan_abi_tbd.txt

```diff
@@ -8,3 +8,15 @@ __asan_on_error
 __asan_print_accumulated_stats
 __asan_set_death_callback
 __asan_update_allocation_context
+__asan_describe_address
+__asan_get_alloc_stack
+__asan_get_report_access_size
+__asan_get_report_access_type
+__asan_get_report_address
+__asan_get_report_bp
+__asan_get_report_description
+__asan_get_report_pc
+__asan_get_report_sp
+__asan_report_error
```

Comment:
I am not sure if we call __asan_report_error from the instrumented program. @rsundahl Can you please verify this

Reply:
Is this a "text-based stub library (`.tbd`)" file or just a "to be defined", list of symbols we need to figure out what to do with?
If it's the former, what/where is this consumed?

Reply:
The list of above is the list of entrypoints that are required by the llvm instrumentation to be provided by the runtime. The test looking for the symbols is: AddressSanitizerABI-arm64-darwin :: TestCases/Darwin/llvm_interface_symbols.cpp. It is likely true that __asan_report_error may not be used by us @usama54321, but it seems to be used by Linux and Windows platforms but it looks like the test is making an unqualified requirements of all platforms that the runtime includes the symbols that it does.